test-requirements.txt

```diff
@@ -1,3 +1,3 @@
-ansible==2.7.5.0
-molecule==2.19.0
-docker==3.3.0
\ No newline at end of file
+ansible>=2.7.8,<2.8
```

Comment:
This change implies that `meta/main.yml` and `Readme.md` should be changed too

Reply:
Done